test/fixture_test.js

```diff
@@ -140,6 +140,26 @@ test('dynamic fixtures', function () {
 		});
 });
 
+test('dynamic fixtures return promises', function () {
+	stop();
+	fixture.delay = 10;
+	fixture('something', function () {
+		return Promise.resolve([{
+			sweet: 'ness'
+		}]);
+	});
+	$.ajax({
+		url: 'something',
+		dataType: 'json'
+	})
+		.done(function (data) {
+			equal(data[0].sweet, 'ness', 'can.get works');
+			start();
+		}).catch(function(err){
+			debugger;
```

Comment:
Should this be removed / replaced with `errorCallback` (and use `then` instead of `done`)?

Reply:
TBH, I just copied the above test and made it return a promise. In this case, I would probably just use `.then` and take out the error catching all together. the test running can handle that =]